### ISSUE TITLE
Sales Data Download Page Accessibility Fixes: contrast, table names, svgs

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3250]
Found vis trusted tester: 

- Links in the process diagram fail contrast requirements (Alex will provide the hex number to use)
- Tables need a name (TC table) 
- decorative svgs need to be hidden from screen readers using an aria label

Most of the svgs are already properly hidden - the only ones that are not are the three in the downloads section: <img width="807" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXRoQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--d88451feafe3b43b72886ba04be3a4fbb295412a/image.png" alt="image.png" />

**Comment by greensomes** [comment]
There are [some fixes ready here](https://nrrd-preview.app.cloud.gov/sites/3250-sales-data-download-page-accessibility-fixes-contrast-table-names-svgs/downloads/federal-sales/).

- Link colors in the process diagram are updated
- Link default/hover stying has been updated to match other links 
- Decorative SVGs have been assigned the aria-hidden attribute
- The aria-label attribute has been applied to the layout tables used in the process diagram

There will be more to come!

**Comment by greensomes** [comment]
The aria-label attribute has been applied to the transaction codes table as well.

**Comment by AlexandracmONRR** [comment]
@greensomes - there is one change that i suggested to the hover state in issue #3196 since we changes the color of the button text in the process diagram. i put pictures in the issue but let me know if you have any questions.
